src/_impl/RequestManager.ts

```diff
@@ -1,11 +1,14 @@
-import { RequestManager, RequestManagerInfo, Request, Response } from ".."
+import { RequestManager, RequestManagerInfo, Request, Response, Cookie } from ".."
 
-//@ts-ignore
 import axios, { Method } from 'axios'
+import { CookieJar } from 'tough-cookie';
+import { HttpCookieAgent, HttpsCookieAgent } from 'http-cookie-agent/http';
 
 const _global = global as any
 
 _global.createRequestManager = function (info: RequestManagerInfo): RequestManager {
+    const jar = new CookieJar();
```

Comment:
Probably worth checking - does `getAllCookies` return all cookies set over the life of the `RequestManager` (which is what this implementation does), or just from the latest request?